Libraries/NavigationExperimental/NavigationPropTypes.js

```diff
@@ -59,14 +59,14 @@ const scene = PropTypes.shape({
 });
 
 /* NavigationSceneRendererProps */
-const SceneRenderer = {
+const SceneRenderer = PropTypes.shape({
```

Comment:
`SceneRenderer` is used as an alias for common props that could be used to render a scene. 

So you can easily define a scene like this:

```
class MyScene extends React.Component {
  render() {
    return (
      <View>
        <Text>{this.props.blah}</Text>
        <Text>{this.props.scene.key}</Text>
      </View>
    );
  }
}

MyScene.propTypes = {
  ...NavigationPropTypes.SceneRenderer,
  blah: PropTypes.string,
};
```

I don't understand why would to turn SceneRenderer  into an PropTypes.shape.